cmd/main.go

```diff
@@ -68,7 +72,7 @@ func main() {
 	rootCmd.AddCommand(editCmd)
 	rootCmd.AddCommand(initCmd)
 	rootCmd.AddCommand(runCmd)
-	rootCmd.AddCommand(upgradeCmd)
+	//rootCmd.AddCommand(upgradeCmd)
```

Comment:
Should we remove this comment since we won't be adding an upgrade command?

Reply:
it technically won't run, i thought i'd leave just incase, but its toggled off